src/components/Search/FrameworkDropdown.tsx

```diff
@@ -116,7 +117,7 @@ function FrameworkDropdown({ view, mode }: FrameworkDropdownProps) {
           onChange={(e) => void handleFrameworkSelect(e)}
           name='Framework'
         >
-          {Object.entries(frameworkMap).map(([id, name]) => (
+          {sortFrameworks(frameworkMap).map(([id, name]) => (
```

Comment:
can we please compute this outside the component ? It will never change because frameworkMap will never change. It makes sense to keep it local to this file (as opposed to have it in the `constants` file) because it's used here only, but it makes no sense to compute this for every render.